new-site/src/docs/elements/components/stepper/customisation.mdx

```diff
@@ -0,0 +1,117 @@
+---
+slug: '/elements/components/stepper/customisation'
+title: 'Stepper - Customisation'
+---
+
+import { Stepper } from 'hds-react';
+
+## Customisation
+
+### Customisation properties
+You can use the `style` property to customise the component. See all available theme variables in the table below.
+
+| Theme variable                        | Description                                 |
+| ------------------------------------- | ------------------------------------------- |
+| `--hds-not-selected-step-label-color` | Colour of a non-selected step label.        |
+| `--hds-step-background-color`         | Background colour of a step circle.         |
+| `--hds-step-content-color`            | Colour of elements inside a step.           |
+| `--hds-stepper-background-color`      | Background colour of the stepper container. |
+| `--hds-stepper-color`                 | Highlight colour of the stepper.            |
+| `--hds-stepper-disabled-color`        | Colour of a disabled step.                  |
+| `--hds-stepper-focus-border-color`    | Colour of a step focus border.              |
+
+### Customisation example
```

Comment:
The customisation example does not seem to work: pressing next button does nothing

Reply:
The example is probably missing the state.